docs/languages/r.md

```diff
@@ -0,0 +1,92 @@
+---
+Order: 13
+Area: languages
+TOCTitle: R
+ContentId: d7ec8e7c-de5e-42b3-86df-a48660f1f6e1
+PageTitle: R in Visual Studio Code
+DateApproved: 03/19/2022
+MetaDescription: Learn about working with the R programming language in Visual Studio Code.
+---
+
+# R in Visual Studio Code
+
+The [R programming language](https://www.r-project.org/) is a dynamic language built for statistical computing and graphics. R is commonly used in statistical analysis, scientific computing, machine learning, and data visualization.
+
+The [R extension](https://marketplace.visualstudio.com/items?itemName=Ikuyadeu.r) for Visual Studio Code supports extended
+syntax highlighting, code completion, linting, formatting, interacting with R terminals, viewing data, plots, workspace variables, help pages, managing packages and working with [R Markdown](https://github.com/REditorSupport/vscode-R/wiki/R-Markdown) documents.
+
+![Overview](images/r/overview.png)
+
+## Getting started
+
+1. [Install R](https://cloud.r-project.org/) (>= 3.4.0) for your platform. For Windows users, Writing R Path to the registry is recommended in the installation.
```

Comment:
Are you able to clarify "Writing R Path..." for new users? Does mean the systems path? ☺️

Reply:
Does it sound better:

[Install R](https://cloud.r-project.org/) (>= 3.4.0) for your platform. For Windows users, *Save version number in registry* is recommended in the installation so that the R extension could find the R executable automatically.

It is actually a checkbox in the R installer and is checked by default. 

![R installer](https://user-images.githubusercontent.com/4662568/76700772-ca94ee00-66f5-11ea-97bc-f89afeaf1bd3.png)

Reply:
Thanks for the clarification!